CMakeLists.txt

```diff
@@ -93,6 +111,9 @@ endif()
 # ----- MPI
 find_package(MPI REQUIRED)
 
+# ----- OpenMP
+find_package(OpenMP REQUIRED)
+
```

Comment:
I think that the if statement is missing...

Reply:
Indeed, I've just pushed a fix for that. Please have another look.